2019-May.html

```diff
@@ -0,0 +1,293 @@
+                    <p>
+                        <img src="img/checkin-to-win.png" alt="Check-in to win"/>
+                    </p>
+
+                    <a href="http://raffles.amsterdamphp.nl">http://raffles.amsterdamphp.nl</a>
+                </section>
+<!--
+                <section>
+                    <h1 class="fancy">Lightning talk!</h1>
+
+                    <h4>Shared understanding: better software through meaningful conversations</h3>
+                    <h4>Emmerson Siqueira</h3>
+                </section> -->
+
+                <section>
+                    <h1>Tonight's Speaker</h1>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <img id="speaker-face" class="speaker" src="" alt="">
+                        </div>
+
+                        <div>
+                            <h3 id="speaker-name" class="fancy"></h3>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="fancy">Rating</h1>
+
+                    <div class="two-columns">
+                        <div>
+                            <img class="meetup" src="img/meetup-large.png" alt="Meetup">
+                            <h2 class="fancy">The Venue</h2>
+                        </div>
+
+                        <div>
+                            <img class="white-background" src="img/joindin.png" alt="JoindIn">
+                            <h2 class="fancy">The Speakers</h2>
+                            <a id="joind-in" href="https://joind.in/talk/c9a87"></a>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <img width="600" src="img/conference-logos/laracon-eu-2018.svg">
+                    <h4>May 23th</h4>
+                    <h4>Madrid, Spain</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Dutch PHP Conference</h1>
+                    <img width="600" src="img/conference-logos/dpc-2018.png">
+                    <h4>June 6th-8th</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
+                <section data-background="img/conference-logos/wecamp-background.jpg">
+                    <h1 class="slide fancy">WeCamp</h1>
+                    <img width="600" src="img/conference-logos/wecamp-2019.png" style="box-shadow: none;">
+                    <h4>August 20th-24th</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
+                <section>
+                    <img width="600" src="img/conference-logos/laracon-eu-2018.svg">
+                    <h4>August 28th-30th</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
```

Comment:
https://2019.webcampzg.org is happening on the 11th and 12th of October and the CfP is open for the rest of the month